src/demo.html

```diff
@@ -67,7 +67,7 @@ <h1>MicroPython-micro:bit simulator example embedding</h1>
         <div class="simulator column">
           <iframe
             id="simulator"
-            src="simulator.html"
+            src="simulator.html?color=%236c4bc1"
```

Comment:
Let's not use the micro:bit brand colour in this demo. blue? Or whatever 😄 

Might be nice to document it in README.md too.

Reply:
[`c577b9f`](https://github.com/microbit-foundation/micropython-microbit-v2-simulator/pull/23/commits/c577b9f70b00036f292e34ed99f130d058d8540b) & README.md updated here [`546d5c7`](https://github.com/microbit-foundation/micropython-microbit-v2-simulator/pull/23/commits/546d5c7b6bcb876d35765bcc8ea4015463b2bc66).